services/core-api/app/api/now_applications/resources/now_application_resource.py

```diff
@@ -71,6 +73,17 @@ def put(self, application_guid):
 
         data = request.json
 
+        verified_by_user_date = data.get('verified_by_user_date', None)
```

Comment:
I am not commenting on this to change it more just to start a convo but I think we should standardize where possible between data.json and the reqparser.